test/lit/passes/Oz.wast

```diff
@@ -58,7 +58,7 @@
   ;; CHECK:      (func $8 (; has Stack IR ;) (param $0 i32) (param $1 i32) (param $2 i32) (param $3 i32) (result i32)
   ;; CHECK-NEXT:  (i32.store
   ;; CHECK-NEXT:   (local.tee $0
-  ;; CHECK-NEXT:    (local.tee $1
+  ;; CHECK-NEXT:    (local.tee $0
```

Comment:
Looks like this:
```
(local.tee $0
  (local.tee $0
    ...
  )
 )
```
Also could be optimized as duplicate (redundant) operation

Reply:
@MaxGraey Yes, see the TODO here: https://github.com/WebAssembly/binaryen/pull/4314/files#diff-4b82a7f223b291ea9808e718d45e91593dc75d9bf684d42386e0f0f480132a16R518 I didn't want to add this in this PR to keep it smaller, but that's a followup.

Reply:
Thanks, now I see